sklearn/_fast_gradient_boosting/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""This module implements the 'fast' gradient boosting estimators.
+
+The implementation is a port from pygbm which is itself strongly inspired
+from LightGBM.
+"""
+from .gradient_boosting import HistGradientBoostingClassifier
+from .gradient_boosting import HistGradientBoostingRegressor
+
+__all__ = ["HistGradientBoostingClassifier", "HistGradientBoostingRegressor"]
```

Comment:
do we want to have `__all__` in `sklearn._....`?

Reply:
I think we need it at least to avoid flake8 warnings:

```
Running flake8 on the diff in the range 7639886fa..bc0d80585 (155 commit(s)):
--------------------------------------------------------------------------------
sklearn/_fast_gradient_boosting/__init__.py:6:1: F401 '.gradient_boosting.HistGradientBoostingClassifier' imported but unused
from .gradient_boosting import HistGradientBoostingClassifier
^
sklearn/_fast_gradient_boosting/__init__.py:7:1: F401 '.gradient_boosting.HistGradientBoostingRegressor' imported but unused
from .gradient_boosting import HistGradientBoostingRegressor
^
```

To bypass this we could instead import the estimators in `experimental/__init__.py` as such:

```
from .._fast_gradient_boosting.gradient_boosting import HistGradientBoostingClassifier
from .._fast_gradient_boosting.gradient_boosting import HistGradientBoostingRegressor
```

WDYT?

Reply:
I like the alternative, but no hard feelings. Although I'm not sure how the modern IDEs would interpret things in the `__all__` of a submodule starting with an underscore. I guess it's safer not to have them and use the alternative.